src/Samples/Sample.AspNetCore/appsettings.json

```diff
@@ -9,10 +9,10 @@
   "AllowedHosts": "*",
 
   "PayeeInfo": {
-    "PayeeId": "91a4c8e0-72ac-425c-a687-856706f9e9a1"
+    "PayeeId": "c60d3e0b-bad8-40fb-b02b-af75fca9702c"
```

Comment:
Wouldn't it be better to have a `PayeeId` with the value of `Guid.Empty` and do a check for that on `Startup` so we can fail with a user friendly error message that the Sample project needs to be properly configured as per the `README`?